src/main/java/mate/academy/entity/Book.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.entity;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private int id;
+    private String title;
+    private BigDecimal price;
+
+    public void setId(int id) {
```

Comment:
The `setId` method should accept a `Long` type parameter to match the task requirements.